src/cli/Show.cpp

```diff
@@ -78,10 +85,19 @@ int Show::showEntry(Database* database, QString entryPath)
         return EXIT_FAILURE;
     }
 
-    outputTextStream << "   title: " << entry->title() << endl;
-    outputTextStream << "username: " << entry->username() << endl;
-    outputTextStream << "password: " << entry->password() << endl;
-    outputTextStream << "     URL: " << entry->url() << endl;
-    outputTextStream << "   Notes: " << entry->notes() << endl;
+    // If no attributes specified, output the default attribute set.
+    bool showAttributeNames = attributes.isEmpty();
+    if (attributes.isEmpty()) {
+        attributes = EntryAttributes::DefaultAttributes;
+    }
+
+    // Iterate over the attributes and output them line-by-line.
+    for (QString attribute : attributes) {
+        if (showAttributeNames) {
+            outputTextStream << attribute << ": ";
+        }
+        outputTextStream << entry->attributes()->value(attribute) << endl;
```

Comment:
@cyphar what's the behaviour when that attribute does not exist?

Reply:
At the moment it will output nothing I believe, but we should be erroring out and doing a `return EXIT_FAILURE`.